contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -11,14 +11,23 @@ toc::[]
 == Modularization backstory
 OpenShift docs are getting modularized, starting from OpenShift 4.0.
 All existing content will be completely replaced with content that is based on user stories and
-complies with the modularization guidelines. All future content must support a user story.
+complies with the modularization guidelines. All future content must both
```

Comment:
@kalexand-rh Do we want a link to the modularization guidelines here, so that people know that we are providing the guidelines? Might be off-putting to mention the guidelines with no indication of what they are.